web/index.html

```diff
@@ -1,519 +1,448 @@
+                                    <p style="color: #666; padding: 0 1em">
+                                        Use a unified API to enable your HPC application to run virtually anywhere.
+                                        <span class="psi-j-font">PSI/J</span> automatically translates abstract job
+                                        specifications into concrete scripts and commands to send to the scheduler.
+                                        <span class="psi-j-font">PSI/J</span> is tested on a wide variety of clusters.
+                                    </p>
+                                </v-card>
+                            </v-col>
+                            <v-col justify="center">
+                                <v-card align="center">
+                                    <div class="icon-logo"><i class="fas fa-terminal" style="color: #DD3D5A"></i></div>
+                                    <h3><span class="psi-j-font">PSI/J</span> runs entirely in user space</h3>
+
+                                    <p style="color: #666; padding: 0 1em">
+                                        There is no need to wait for infrequent deployment cycles. The HPC world
+                                        tends to be rather dynamic and the ability to quickly integrate changes
+                                        prompted by experimental changes in the cluster environment is essential.
+                                    </p>
+                                </v-card>
+                            </v-col>
+                            <v-col justify="center">
+                                <v-card align="center">
+                                    <div class="icon-logo"><i class="fas fa-puzzle-piece" style="color: #F15A3D"></i>
+                                    </div>
+                                    <h3>Use built-in or community contributed plugins</h3>
+
+                                    <p style="color: #666; padding: 0 1em">
+                                        Let's be realistic. It is virtually impossible for a single entity to
+                                        provide stable and tested adapters to all clusters and schedulers. That is
+                                        why <span class="psi-j-font">PSI/J</span> enables and encourages community
+                                        contributions to scheduler adapters, testbeds, and specific cluster
+                                        knowledge.
+                                    </p>
+                                </v-card>
+                            </v-col>
+
+                            <v-col justify="center">
+                                <v-card align="center">
+                                    <div class="icon-logo"><i class="fas fa-save" style="color: #FDA214"></i></div>
+                                    <h3><span class="psi-j-font">PSI/J</span> has a rich HPC legacy</h3>
```

Comment:
Not really a change you made in this PR, but I wonder if "legacy" should be "ancestry". I think "legacy" suggests that things have been built off of it, but really it's that PSI/J is built off of older things.